mpp/base/mpp_frame.cpp

```diff
@@ -144,6 +144,8 @@ MPP_RET mpp_frame_info_cmp(MppFrame frame0, MppFrame frame1)
         (f0->height             == f1->height) &&
         (f0->hor_stride         == f1->hor_stride) &&
         (f0->ver_stride         == f1->ver_stride) &&
+        (f0->time_scale         == f1->time_scale) &&
```

Comment:
There is no necessary to add info change test. For framerate change will reflect on pts.
The info change judgement will report resolution / bitdepth / format changes which will affect buffer allocation and display setup.

Reply:
Yes I had doubts about where to put the information and how to have the parser notify the app when those values change.  What do you think would be the best way ?

Reply:
In fact, we're already in the process moving away from PTS-based detection of framerate changes, and on our side the need for this patch is temporary.  If you feel it's not the right place for the feature, let's just close this PR, and we'll just keep this patch on our side until our need for it is gone :)